assets/js/item_details.js

```diff
@@ -149,6 +152,107 @@ $("#add-image-input").change(function (e) {
 });
 
 
+$(".deleteRow").click((e) => {
+    if ($(e.target).siblings(".item_quantity").val() > 0) {
+        return
+    }
+    let deletedLocation = $(e.target).parent().parent().parent()
```

Comment:
This is less likely to break if you use .closest(selector). That way a single extra div won't break this

Reply:
closest("tr") for first transitive parent table row?

Reply:
I'm not sure which element you are referring to by .parent().parent().parent() but maybe give it an id (if unique) or a class name (if not)?

Reply:
That's actually the other benefit, from code people will know what you are referring to and what you were referring to when things break.